ci-docker-base/Dockerfile.cilint-clang-tidy

```diff
@@ -25,8 +25,11 @@ RUN cd ./clang-tidy-checks && \
     cp -r llvm-project/build . && \
     rm -rf llvm-project
 
-# Link clang-tidy to custom build
-RUN ln -s $(pwd)/clang-tidy-checks/build/bin/clang-tidy /usr/bin/clang-tidy
+# Copy binary to /usr/bin
+#
+# We are copying the binary (instead of symlinking it) to avoid header file
+# conflicts between the installed version of clang and our custom build
+RUN cp /clang-tidy-checks/build/bin/clang-tidy /usr/bin/clang-tidy
```

Comment:
Why no more `$(pwd)/clang-tidy-checks/build/bin/clang-tidy` but instead just `/clang-tidy-checks/build/bin/clang-tidy`?

Reply:
Because we are already in `/`, and so `$(pwd)` is redundant